content/en/serverless/google_cloud_run/_index.md

```diff
@@ -2,201 +2,234 @@
-COPY --from=datadog/serverless-init:beta4 /datadog-init /app/datadog-init
+COPY --from=datadog/serverless-init /datadog-init /app/datadog-init
 ENTRYPOINT ["/app/datadog-init"]
 CMD ["./mvnw", "spring-boot:run"] (adapt this line to your needs)
 
 ```
 
-See [Tracing Java Applications][1] for detailed instructions. [Sample code for a simple Java application][2].
+[Sample code for a simple Java application](https://github.com/DataDog/crpb/tree/main/java).
 
-[1]: /tracing/setup_overview/setup/java/?tabs=containers
-[2]: https://github.com/DataDog/crpb/tree/main/java
 {{< /programming-lang >}}
-
 {{< programming-lang lang="dotnet" >}}
+
 ```
-COPY --from=datadog/serverless-init:beta4 /datadog-init /app/datadog-init
+COPY --from=datadog/serverless-init /datadog-init /app/datadog-init
 ENTRYPOINT ["/app/datadog-init"]
 CMD ["dotnet", "helloworld.dll"] (adapt this line to your needs)
 
 ```
 
-See [Tracing .NET Applications][1] for detailed instructions. [Sample code for a simple .NET application][2].
+[Sample code for a simple .NET application](https://github.com/DataDog/crpb/tree/main/dotnet).
 
-[1]: /tracing/trace_collection/dd_libraries/dotnet-core?tab=containers
-[2]: https://github.com/DataDog/crpb/tree/main/dotnet
 {{< /programming-lang >}}
 {{< programming-lang lang="ruby" >}}
+
 ```
-COPY --from=datadog/serverless-init:beta4 /datadog-init /app/datadog-init
+COPY --from=datadog/serverless-init /datadog-init /app/datadog-init
 ENTRYPOINT ["/app/datadog-init"]
 CMD ["rails", "server", "-b", "0.0.0.0"] (adapt this line to your needs)
 
 ```
 
-See [Tracing Ruby Applications][1] for detailed instructions. [Sample code for a simple Ruby application][2].
+[Sample code for a simple Ruby application](https://github.com/DataDog/crpb/tree/main/ruby-on-rails).
 
-[1]: /tracing/trace_collection/dd_libraries/ruby/
-[2]: https://github.com/DataDog/crpb/tree/main/ruby-on-rails
 {{< /programming-lang >}}
 {{< /programming-lang-wrapper >}}
 
-#### Troubleshooting
-This integration depends on your runtime having a full SSL implementation. If you are using a slim image for Node, you may need to add the following command to your Dockerfile to include certificates.
+#### Configure the Datadog Agent using buildpack
 
-```
-RUN apt-get update && apt-get install -y ca-certificates
-```
+[`Pack Buildpacks`][4] provide a convenient way to package your container without using a Dockerfile. This example will use the GCP container registry and Datadog serverless buildpack. Build your application by running the following command:
 
-#### Build with the Datadog buildpack
-
-1. Use [`pack`][4] to build your application by running the following:
-   ```
+   ```shell
    pack build --builder=gcr.io/buildpacks/builder \
    --buildpack from=builder \
-   --buildpack datadog/serverless-buildpack:beta4 \
+   --buildpack datadog/serverless-buildpack \
    gcr.io/YOUR_PROJECT/YOUR_APP_NAME
-
    ```
 
-   **Note**: Not compatible with Alpine.
+**Note**: Not compatible with Alpine.
```

Comment:
Since our buildpack doesn't install the tracer, we still need to link user to doc like this https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/python?tab=containers#instrument-your-application

Reply:
That section exists only for Java, Python, Nodejs, Ruby, dotnet.
Golang part is missing, but there is something here https://docs.datadoghq.com/serverless/installation/go/?tab=serverlessframework#install-the-datadog-lambda-library

Reply:
I added a generic tracing link together with some other changes